codespell_lib/__main__.py

```diff
@@ -0,0 +1,4 @@
+from ._codespell import _script_main
+
+if __name__ == '__main__':
```

Comment:
You can also add the __name__ stuff in _codespell.py (which is what I'd been doing just now), but perhaps that has path issues which this doesn't.

Reply:
I don't see a need to do it in this PR (seems like a separate issue) or really have it in the codebase now that we can just do `python -m codespell_lib ...`

Reply:
Yeah I suspect the module is nicer than loading some random file. It was more just a comment it's possible instead of needing the __main__, your options is probably better though.